Integrations/integration-Tenable_io.yml

```diff
@@ -0,0 +1,673 @@
+        valid_scans = filter(lambda x: x is not None, scan_entries)
+        invalid_scans = [k for k,v in zip(response['scans'], scan_entries) if v is None]
+        res = [get_entry_for_object('Tenable.io - List of Scans', 'TenableIO.Scan', replace_keys(valid_scans), GET_SCANS_HEADERS)]
+        if invalid_scans:
+            res.append(get_entry_for_object('Inactive Web Applications Scans - Renew WAS license to use these scans', 'TenableIO.Scan', replace_keys(invalid_scans), GET_SCANS_HEADERS, remove_null=True))
+        return res
+
+    def launch_scan(scan_id, targets):
+        scan_info = send_scan_request(scan_id)['info']
+        if not targets:
+            targets = scan_info.get('targets', '')
+        res = send_scan_request(scan_id, 'launch', 'POST', alt_targets=targets)
+        res.update({
+            'id': scan_id,
+            'targets': targets
+        })
+
+        return get_entry_for_object('The requested scan was launched successfully', 'TenableIO.Scan', replace_keys(res, LAUNCH_SCAN_NAMES_MAP), LAUNCH_SCAN_HEADERS)
+
+    def get_report(scan_id, info, detailed):
+        results = []
+        scan_details = send_scan_request(scan_id)
+
+        if info == 'yes':
+            scan_details['info']['id'] = scan_id
+            scan_details['info'] = replace_keys(scan_details['info'])
+            results.append(get_entry_for_object('Scan basic info', 'TenableIO.Scan', scan_details['info'], SCAN_REPORT_INFO_HEADERS))
+
+        vuln_info, vulns_not_found = get_vuln_info(scan_details['vulnerabilities'])
+        vuln_info = replace_keys(vuln_info)
+        results.append(get_entry_for_object('Vulnerabilities', 'TenableIO.Vulnerabilities', vuln_info, SCAN_REPORT_VULNERABILITIES_HEADERS))
+        if len(vulns_not_found) > 0:
+            vulns_not_found = replace_keys(vulns_not_found)
+            results.append(get_entry_for_object('Vulnerabilities - Missing From Workbench', 'TenableIO.Vulnerabilities', vulns_not_found, SCAN_REPORT_VULNERABILITIES_HEADERS, True))
+
+        if detailed == 'yes':
+            assets = replace_keys(scan_details['hosts'] + scan_details['comphosts'])
+            results.append(get_entry_for_object('Assets', 'TenableIO.Assets', assets, SCAN_REPORT_HOSTS_HEADERS))
+            if 'remediations' in scan_details and 'remediations' in scan_details['remediations'] and len(scan_details['remediations']['remediations']) > 0:
+                remediations = replace_keys(scan_details['remediations']['remediations'], REMEDIATIONS_NAMES_MAP)
+                results.append(get_entry_for_object('Remediations', 'TenableIO.Remediations', remediations, SCAN_REPORT_REMEDIATIONS_HEADERS))
+        return results
+
+    def get_vulnerability_details(plugin_id, date_range):
+        info = send_vuln_details_request(plugin_id, date_range)
+        if 'error' in info:
+            return { 'Type': entryTypes['error'], 'ContentsFormat': formats['text'], 'Contents': info['error'] }
```

Comment:
format the dictionary. 
`{ 'Type': entryTypes['error'], 'ContentsFormat': formats['text']`
=>
```
{
        'Type': entryTypes['error'],
.....
}
```

Reply:
same use return_error